docs/search-views/fulltext-search/concepts/fulltext-search-setup.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 10
+title: Fulltext Search View Setup
+---
+
+Fulltext search views in Macrometa's Global Data Network (GDN) provide a powerful tool for carrying out efficient, text-based searches on your collections. To get the most from fulltext search views, it's essential to correctly set up your collections and define how you want your data to be indexed and searched.
+
+## Linking Collections
+
+When you create a search view, you choose which collections to link it to. A link transforms your collections into searchable data sources, creating a one-way data flow from a collection to a search view. By establishing multiple links from different collections to a single search view, you can perform a single search across multiple collections.
+
+You can also link more than one search view to a particular collection. Linking one collection to multiple search views allows you to apply different search strategies to the same data set. This flexibility is invaluable when dealing with complex applications with varied search requirements.
+
+## Handling Data Structures
+
+You can link key-value stores, document stores, and graph edge collections to search views. This setup allows your data to be treated as both flat and interconnected structures, supporting both straightforward attribute-based searches and more complex searches involving relationships between data entities.
```

Comment:
Do we want to mention exception in `key-value stores, document stores, and graph edge collections`? If yes, then KV blob might be exception that @amolth-macrometa may confirm.

Reply:
@maheshw2, see question above

Reply:
@diana-macrometa yes, that is my understanding but it needs to be confirmed by @amolth-macrometa

Reply:
Correct; blob-based collections do not support search views of any kind.

Reply:
Is the note I added below sufficient, or do I need more clarification?